src/docs/admin/references/NodeSourceInfrastructureReference.adoc

```diff
@@ -588,7 +588,96 @@ The next four parameters are optional, and apply to advanced Azure endpoint conf
+- *osType:* A choice between _linux_ and _windows_. Default value is _linux_.
+- *Image:* You can either provide a custom Azure image, or select one of the available standard ones. To provide a custom image use the format _{resourceGroupName}:{imageName}_. If you prefer to use a generic image, you can choose from the following lists (provided by Azure API):
++
+|===
+|Linux |Windows
+
+a|- UBUNTU_SERVER_14_04_LTS
+  - UBUNTU_SERVER_16_04_LTS
+  - DEBIAN_8 +*+ _default value_ +*+
+  - CENTOS_7_2
+
+a|- WINDOWS_SERVER_2008_R2_SP1
+  - WINDOWS_SERVER_2012_DATACENTER
+  - WINDOWS_SERVER_2012_R2_DATACENTER
+|===
+
+- *sshPublicKey:* Optional parameter. In case of Linux, use it to provide a public RSA key to connect to your VMs via SSH. In case of Windows, use it to define a password to connect to your VMs via Remote Desktop. In both cases, the default user is _activeeon_. If the field is left blank a password will be automatically generated and written in the Scheduler log.
+- *targetNetwork:* Optional parameter. Use it to define a specific Azure Virtual Network subnet for your Scale Set. If defined, the format should be _{resourceGroupName}:{network}:{subnet}_, e.g. _myGroup:myVNet:default_. If left unset, a new Virtual Network and subnet will be created inside the Resource Group.
+- *region:* Define the Azure region where the Resource Group and the Scale Set will be created. For a list of the regions supported by your subscription, you can use the https://docs.microsoft.com/en-us/cli/azure/account?view=azure-cli-latest#az-account-list-locations[CLI command] `az account list-locations -o table`, the right name to use is the one in the column _name_. The region name should be in lowercase without spaces, e.g. for the region _West US 2_ use _westus2_. Default value is _westeurope_.
+- *rmUrl:* Optional parameter. Provide your server's URL or public IP (e.g. try.activeeon.com). If left unset, the default Resource Manager URL will be fetched from the system properties.
+- *rmCredentials:* Optional parameter. Provide credentials to connect to the Resource Manager. If left blank, the credentials of the logged user will be used.
+- *deploymentTimeout:* Delay before the Resource Manager can declare a deploying node as lost. To be set in _minutes_. You can set it to a reasonable time after which you expect the node to be running or marked as lost otherwise (e.g. _10_ minutes). Default value is _120_.
+- *cleaningDelay:* Periodical cleaning delay, expressed in _seconds_. Every _X_ seconds the Resource Manager will actually remove decommissioned instances from the Scale Set. Default value is _60_.
```

Comment:
Bis repetita

Reply:
You mean about "remove decommissioned"? I agree it's not clear (but it is actually what I meant). Let me rephrase it.

Reply:
My point was not about about `decommissioned` but about the unit as well. Well, it's my mistake too here.